themes/scss/widgets/tooltip.scss

```diff
@@ -14,7 +11,7 @@
   padding: 5px;
   box-sizing: border-box;
   border-radius: $baseSize / 2;
-  font: $sFontSize-smaller 'Open Sans';
+  font: $sFontSize-small 'Open Sans';
```

Comment:
Name of variable `sFontSize-small` should be written in all lowercase letters with hyphens instead of underscores<br>Prefer double-quoted strings